plugins/netskope/icon_netskope/connection/schema.py

```diff
@@ -0,0 +1,66 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Input:
+    API_KEY_V1 = "api_key_v1"
+    API_KEY_V2 = "api_key_v2"
+    TENANT = "tenant"
+    
+
+class ConnectionSchema(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "api_key_v1": {
+      "$ref": "#/definitions/credential_secret_key",
+      "title": "API Key v1",
+      "description": "Netskope authorization token for v1 API",
+      "order": 2
+    },
+    "api_key_v2": {
+      "$ref": "#/definitions/credential_secret_key",
+      "title": "API Key v2",
+      "description": "Netskope authorization token for v2 API",
+      "order": 3
+    },
+    "tenant": {
+      "type": "string",
+      "title": "Tenant",
+      "description": "Owner's name that is included in URL",
+      "order": 1
+    }
+  },
+  "required": [
+    "api_key_v1",
```

Comment:
Why do the user need both the keys?

Reply:
At the beginning we have discussed that with Mike. There are two apis v1 and v2 which has two different keys. Only one action uses api v1 key (Update File Hash List). The decision was to leave both of them required in connection.